packages/did-auth-siop-op-authenticator/__tests__/shared/mockedData.ts

```diff
@@ -115,7 +119,7 @@ export const vcs = [
 export const vpSingle = [
   {
     format: 'ldp_vp',
-    location: 'id_token',
+    location: 'presentation_definition',
```

Comment:
This doesn't make sense. We are in the VP here. You cannot place a VP in a definition

Reply:
fixed